.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false
+before_install:
+  # For tests running hg command
+  # set up username
+  - echo '[ui]' > ~/.hgrc
+  - echo 'username = anybox-test' >> ~/.hgrc
+
+language: python
+
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - pip install .
+  - pip install coveralls
+  - pip install flake8
+  - pip install bzr
```

Comment:
I'm not an expert about travis configuration, I guess we need add some requirements according documentation or the doc is obsolete?

...
Install nose, flake8 and, optionally, coverage:
```
recipe-env/bin/pip install nose coverage flake8==2.0 \
                   pep8==1.4.6 mccabe==0.2.1 pyflakes==0.7.3
```
...

Maybe we could user a dev requirement file ? Am I wrong?